static/js/admin.js

```diff
@@ -549,6 +549,36 @@ function _setup_page() {
         });
     });
 
+    $(".admin_user_table").on("click", ".open-user-form", function (e) {
+        var email = $(e.currentTarget).data("email");
+        var user_row = $("tr[id='user_" + email + "']");
+        var user_name = user_row.find(".user_name");
+        var form_row = $("tr[id='user_form_" + email + "'");
+        var reset_button = form_row.find(".reset_edit_user");
+        var name_form = form_row.find(".name-setting");
+        var name_input = form_row.find("#full_name");
+
+        user_row.hide();
+        form_row.show();
+
+        function show_user_row() {
+            form_row.hide();
+            user_row.show();
+        }
+
+        reset_button.on("click", function (e) {
+            show_user_row();
+        });
+
+        name_form.ajaxForm({
+            success: function (resp, statusText, xhr, form) {
+                show_user_row();
+                user_name.text(name_input.val());
```

Comment:
So, I think you missed a piece of what I was saying -- we perhaps still want to have a `static/js/server_events.js` handler for the name change event adjusting this table...